CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
-## [v1.2.3]
+## [v2.0.0](https://github.com/nextcloud-libraries/nextcloud-vite-config/tree/v2.0.0)
+### Breaking changes
+This version is Vue3 only.
+For Vue2 use the 1.x versions, 2+ will be providing a common config for apps and libraries built using Vue3.
```

Comment:
It duplicates the package description from readme, which is not about the change.

```suggestion
For Vue 2 use the 1.x version.
```